peripherals/dac/dac_wl.yaml

```
@@ -0,0 +1,93 @@
+DAC:
+  "CR,CR?":
+    "CEN?":
+      Normal: [0, 'DAC Channel X Normal operating mode']
+      Calibration: [1, 'DAC Channel X calibration mode']
+    "DMAUDRIE?":
+      Disabled: [0, "DAC Channel X  DMA Underrun Interrupt disabled"]
+      Enabled: [1, "DAC Channel X DMA Underrun Interrupt enabled"]
+    "DMAEN?":
+      Disabled: [0, "DAC Channel X DMA mode disabled"]
+      Enabled: [1, "DAC Channel X DMA mode enabled"]
+    "MAMP?":
+      amp_1: [0b0000, 'Unmask bit0 of LFSR/ triangle amplitude equal to 1']
+      amp_3: [0b0001, 'Unmask bits[1:0] of LFSR/ triangle amplitude equal to 3']
+      amp_7: [0b0010, 'Unmask bits[2:0] of LFSR/ triangle amplitude equal to 7']
+      amp_15: [0b0011, 'Unmask bits[3:0] of LFSR/ triangle amplitude equal to 15']
+      amp_31: [0b0100, 'Unmask bits[4:0] of LFSR/ triangle amplitude equal to 31']
+      amp_63: [0b0101, 'Unmask bits[5:0] of LFSR/ triangle amplitude equal 63']
+      amp_127: [0b0110, 'Unmask bits[6:0] of LFSR/ triangle amplitude equal to 127']
+      amp_255: [0b0111, 'Unmask bits[7:0] of LFSR/ triangle amplitude equal to 255 ']
+      amp_511: [0b1000, 'Unmask bits[8:0] of LFSR/ triangle amplitude equal to 511 ']
+      amp_1023: [0b1001, 'Unmask bits[9:0] of LFSR/ triangle amplitude equal to 1023 ']
+      amp_2047: [0b1010, 'Unmask bits[10:0] of LFSR/ triangle amplitude equal to 2047 ']
```

Remove tailing space:
```suggestion
      amp_255: [0b0111, 'Unmask bits[7:0] of LFSR/ triangle amplitude equal to 255']
      amp_511: [0b1000, 'Unmask bits[8:0] of LFSR/ triangle amplitude equal to 511']
      amp_1023: [0b1001, 'Unmask bits[9:0] of LFSR/ triangle amplitude equal to 1023']
      amp_2047: [0b1010, 'Unmask bits[10:0] of LFSR/ triangle amplitude equal to 2047']
```